docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -53,8 +53,8 @@ learned about the relationships between the data points to classify new data.
 
 
 [discrete]
-[[dfa-classification-model]]
-===== {classification-cap} models
+[[dfa-classification-algorithm]]
+===== {classification-cap} algorithms
```

Comment:
I wonder if the earlier "Training the {classification} algorithm" is correct now.  Should it be "Training the {classification} model" instead?

Reply:
Right! You train the model. I fixed it.